example/lib/ow_broadcast_helper.dart

```diff
@@ -29,8 +30,9 @@ class OWBroadcastHelper {
       BuildContext context) {
     var pinAuthSubs = initPinAuthenticationSubscriptions(context);
     var fingerprintSubs = initFingerprintSubscriptions(context);
```

Comment:
is fingerprint still needed here?

Reply:
Not required. Will remove in separate PR.